autopush/tests/test_rs_integration.py

```diff
@@ -69,9 +69,11 @@ class MockMegaphoneRequestHandler(BaseHTTPRequestHandler):
     API_PATTERN = re.compile(r'/v1/broadcasts')
     services = {}
     polled = Event()
+    token = uuid.uuid4().hex
```

Comment:
Note that the auth header should be "Bearer sometoken". This patch requires "Bearer " to be specified along side the token in the config.

megaphone sort of has to hardcode "Bearer" in its code I think. I'd lean towards these 2 configs matching. If not, at the very least I'd call this config value a "header" vs a "token" and document it w/ an example. and probably include a "Bearer " on this line as well

(autopush_connection.ini could also probably use an example)